resources/charts/calico/values.yaml

```diff
@@ -0,0 +1,11 @@
+calico:
```

Comment:
This can be merged with previous commit, which adds Calico chart. `values.yaml` is always part of the chart.